src/test/kotlin/com/igorwojda/string/decapitalizeconst/challenge.kt

```diff
@@ -4,7 +4,15 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun decapitalizeConst(str: String): String {
-    TODO("not implemented")
+    val wordList=str.split("_")
```

Comment:
Can you find an implementation in which you chain collection operators so get to the result?

Reply:
return str.split("_").map { it.toLowerCase().capitalize() }.joinToString().replace(", ","").decapitalize()

Reply:
return str.split("_").map { it.toLowerCase().capitalize() }.joinToString().replace(", ","").decapitalize()